### DISCUSSION TITLE
System design

### DISCUSSION THREAD

**Discussion by sonck201** [nashtech-garage/nodejs-msa#27] [Ideas]
# Overview

![System context](https://user-images.githubusercontent.com/8024978/233017181-3d93eb44-6390-4648-92ac-1da988d3795c.png)

![Container context](https://user-images.githubusercontent.com/8024978/233017222-bdc691ba-2ea9-4c8b-be06-69dbd29f5da3.png)

**Comment by sonck201** [comment]
Think about the architecture design
- DDD
- Hexagon
- Union

## Missing container
- [ ] NoSQL

**Comment by sonck201** [comment]
MSA design pattern

**Comment by panoti** [comment]
https://dapr.io/

https://github.com/vietnam-devs/coolstore-microservices
